# **CHI-SQUARE TEST**

**1. State the Hypothesis:**


The null hypothesis(H0):There's No significant association between the type of device purchased and the customer's satisfaction level.

The alternate hypothesis(H1):There's a significant association between the type of device purchased and the customer's satisfaction level.

**2. Compute the Chi-Square Statistic:**

In [1]:
A<-matrix(c(50,70,80,100,60,90,30,50,20,50),nrow = 5,ncol = 2,byrow = TRUE)
colnames(A)<-c("Smart Thermostat","Smart Light")
rownames(A)<-c("Very Satisfied","Satisfied","Neutral","Unsatisfied","Very Unsatisfied")
chisq.test(A)


	Pearson's Chi-squared test

data:  A
X-squared = 5.6382, df = 4, p-value = 0.2278


**3. Determine the Critical Value:**

We got the df as 4 and significance level is 0.05.

So,as per CHI-SQUARE table at 0.05 significance level and df as 4 the critical value is 9.48773.

**4. Make a Decision:**

test statistics=5.6382 < critical value=9.48773
The test statistics is less than the critical value so,we can accept the null hypothesis at the specified significance level.

**5.Conclusion**

Based on the step 4 we can Accept the null hypothesis.We have the probability of null hypothesis being true is 0.2278 which is greater than the significance level 0.05.So, we can
conclude that There's No significant association between the type of device purchased and the customer's satisfaction level.